backend/Tasque.WebAPI/appsettings.json

```diff
@@ -12,9 +12,6 @@
     "Key": "BSA-2022-Tasque.Core"
   },
   "AllowedHosts": "*",
-  "ConnectionStrings": {
```

Comment:
are you sure you want to delete this?

Reply:
Yep. We hold connection strings in the user's secrets. This is just a template.